automathor.py

```
@@ -0,0 +1,205 @@
+    def __repr__(self):
+        """Return a string representation of the context."""
+        return f"{self.filename}:{self.line}\n{self.author_email} {self.datetime}\n{self.commit}\n"
+
+    def render(self) -> str:
+        """Render the context as a markdown file."""
+        template_loader = jinja2.FileSystemLoader(searchpath="./")
+        template_env = jinja2.Environment(loader=template_loader, autoescape=True)
+        template = template_env.get_template("template.jinja2")
+
+        output = template.render(context=self)
+        return output
```

Ruff (0.8.2)

Unnecessary assignment to `output` before `return` statement

Remove unnecessary assignment

(RET504)